src/main/java/de/retest/recheck/review/ignore/io/ErrorHandlingLoader.java

```diff
@@ -0,0 +1,33 @@
+package de.retest.recheck.review.ignore.io;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.recheck.review.ignore.FilterPreserveLineLoader.FilterPreserveLine;
+
+public final class ErrorHandlingLoader implements Loader<FilterPreserveLine> {
```

Comment:
`FilterPreserveLine` is a `Filter`, right? If so, please use `Filter`.

Reply:
This is consistent with `FilterPreserveLineLoader`. Either change both or keep it that way... ;-)